### ISSUE TITLE
[Package Issue]: Microsoft.Office does not have previous versions available 

### ISSUE THREAD

**Issue by lloydjatkinson** [microsoft/winget-pkgs#99827]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Other

### Brief description of your issue

Old versions of this package seem to be completely absent?

![image](https://user-images.githubusercontent.com/5633938/226070796-e70dfdfa-8f0c-4e67-9be5-b085459441bb.png)

Now compare to another package:
![image](https://user-images.githubusercontent.com/5633938/226070827-d1a6a353-064e-40ea-9f9b-bddb4621e085.png)

### Steps to reproduce

Run show command for Microsoft.Office

### Actual behavior

The old versions are not available

### Expected behavior

That other versions are available like other packages. This is essential behaviour when the recurring Office hash version mismatch occurs. We should be able to pick an older version if we cannot use the latest.

https://github.com/microsoft/winget-pkgs/issues?q=Microsoft.Office+label%3AError-Hash-Mismatch

### Environment

```raw
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1413
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

User Settings: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
```

**Comment by ItzLevvie** [comment]
`Microsoft.Office` uses a URL that always points to the latest version which is why there's no previous versions.

As a workaround for hash issues, you can run `winget settings --enable InstallerHashOverride` and pass `--ignore-security-hash` to the `winget upgrade` command.

**Comment by BrandonWanHuanSheng** [comment]
Yes, it is true. Office Update is always point to catalog.update.microsoft.com but sometimes the link is directly passing to a newer version. If you want to know more about the office, Please see https://learn.microsoft.com/en-us/officeupdates/release-notes-microsoft365-apps

**Comment by Trenly** [comment]
Since Office has a URL that always points at the latest version, it isn't possible for older versions to be retained. Since this hasn't changed in over a year, I'm going to close this issue to keep things a bit cleaner. Thank you for your understanding

###### Close with reason: Vanity URL; Expected behavior
